### PR TITLE
[CIVP-10269] make sure matplotlib can always find the matplotlibrc

### PR DESCRIPTION
**Pull request by beckermr**
I don't know the best spot to put this, so I chose `/opt/matplitlibrc/matplotlibrc`. There may be a better way.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,9 @@ Version number changes (major.minor.micro) in this package denote the following:
 - A minor version will increase if one or more packages contained in the Docker image add new, backwards-compatible features, or if a new package is added to the Docker image.
 - A major version will increase if there are any backwards-incompatible changes in any of the packages contained in this Docker image, or any other backwards-incompabile changes in the execution environment.
 
+### Fixed
+- Changed location of `matplotlibrc` to always be found (#6)
```
Comment (stephen-hoover) [inline CHANGELOG.md]: This goes under the [1.1.0] header.

**Review by stephen-hoover — CHANGES_REQUESTED** [review]
From the test failure, it seems that `matplotlib` isn't finding the config file. Was the environment variable input new in v2.0? We have v1.5 in this container.

File: CHANGELOG.md (hunk 1 above)
Comment (beckermr) [reply CHANGELOG.md]: Ahhhh. I did not know if that was tagged yet. Will do.

**Comment by beckermr** [comment]
We should be using v2.0 anyways for matplotlib, right? Why are we shipping outdated packages?

**Comment by beckermr** [comment]
I will check into the config thing.

**Comment by beckermr** [comment]
Yeah this is in the 1.5.3 docs

http://matplotlib.org/1.5.3/users/customizing.html#the-matplotlibrc-file

**Comment by beckermr** [comment]
Well whatever. It is fixed now.

**Comment by stephen-hoover** [comment]
`matplotlib` v2.0 came out in the past couple of weeks. It incremented a major version -- do you know if there's backwards-incompatible changes? If so, it would be a major version increment for this container to include it.

**Review by stephen-hoover — APPROVED** [review]
LGTM!

**Pull request merged**